sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/owaspzapwrapper/util/SystemUtil.java

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.owaspzapwrapper.util;
+
+public class SystemUtil {
+
+    public void waitForMilliseconds(int milliseconds) {
```

Comment:
Please add a JavaDoc comment, explaining, that this class uses `Thread.sleep()` under the hood.